src/components/va-alert/va-alert.css

```diff
@@ -11,6 +11,33 @@
   background-color: unset;
 }
 
+:host([full-width='true']),
+:host([full-width='']) {
+  border-top-style: solid;
+  border-top-width: 10px;
+}
+
+:host([status='warning']) {
+  border-color: var(--color-gold);
+}
+:host([status='info']) {
+  border-color: var(--color-primary-alt-dark);
+}
+:host([status='error']) {
+  border-color: var(--color-secondary-dark);
+}
+:host([status='continue']),
+:host([status='success']) {
+  border-color: var(--color-green);
+}
+
+:host([full-width='']) div.alert,
+:host([full-width='true']) div.alert {
+  border-left: none;
+  max-width: 1000px;
+  margin: 0 auto;
+}
```

Comment:
If we're a `full-width` alert, remove the left border from the inner div(*) and limit the width appropriately.

*I didn't want this PR to get too out of scope and have it change the original implementation of the left border. There may be a reason why the border is on an inner div as opposed to `:host`.